index.html

```diff
@@ -4,16 +4,148 @@
+          <li class="black-items fab fa-twitter gray"><a href="#"></a></li>
+          <li class="black-items"><a href="#">English</a></li>
+          <li class="black-items"><a href="#">My Page</a></li>
+          <li class="black-items"><a href="#">Logout</a></li>
+        </ul>
+      </div>
+      <div class="header-white">
+        <a href="./index.html" class="header-logo" ><img src="./Assets/logo/logo-Nba.PNG" alt="NBA logo" class="nba-logo"/></a>
+          <ul class="menu-bars">
+            <i class="fa-solid fa-bars fa-2x black" onclick="hamburgeOn()"></i>
+          </ul>
+        <nav class="navbar-container">
+          <ul class="navbar-list">
+            <li class="navbar-items"> <a href="./about.html" class="navbar-link">About</a> </li>
+            <li class="navbar-items"> <a href="#" class="navbar-link">Program</a> </li>
+            <li class="navbar-items"> <a href="#" class="navbar-link">Join</a> </li>
+            <li class="navbar-items"> <a href="#" class="navbar-link">Sponsor</a> </li>
+            <li class="navbar-items"> <a href="#" class="navbar-link">News</a> </li>
+            <li class="navbar-items"> <a href="./index.html" class="navbar-link red-border">NBA</a> </li>
+          </ul>
+        </nav>
+      </div>
+    </header> 
+    <main class="main">
+          <!-- Main headline background image -->
+      <img alt="headline main background of capston" src="./Assets/images/background12.PNG" class="background-image">
+          <!--Humbarge Overlay Menu area -->
+      <div class="overlay-menu">
+        <div>
+          <i onclick="hamburgeOff()" class="fa-solid fa-close fa-2x close-hamburg"></i>
+        </div>
+        <nav class="overlay-menu-list" onclick="hamburgeOff()">
+          <a href="./about.html"><p>About</p></a>
+          <a href="#about"><p>Program</p></a>
+          <a href="#contact"><p>Join</p></a>
+          <a href="#contact"><p>Sponsor</p></a>
+          <a href="#contact"><p>News</p></a>
+          <a href="./index.html"><p>NBA</p></a>
+        </nav>
+      </div>
+          <!-- Main headline content area -->
+      <section class="headline-container">
+        <h2 class="hello-world">"Hello! Sharing World"</h2>
+        <h1 class="heading"> NBA - ABU DHABI GAMES 2022 </h1>
+        <p class="heading-details">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Quidem,
+          error? Fugiat omnis sed suscipit deleniti, a rem dolorem at, quisquam
+          aliquid voluptatem nobis quas quo!
+        </p>
+        <div>
+          <p class="program-date">
+            <span class="normal-font">2022.10.06</span>(THU) &
+            <span class="normal-font">08</span>(SAT)
+          </p>
+          <p class="program-location">@ Etihad Arena on Yas Island, Abu Dhabi</p>
+        </div>
+      </section>
     </main>
-    <footer class="footer">
-    </footer>
+              <!-- Program cards Section -->
```

Comment:
Optional
on your hamburger menu, in the mobile version, you should change the pointer to notify users that it is clickable 😄

Reply:
Add change cursor as a pointer on hover